app/src/main/res/values-xhdpi/dimens.xml

```diff
@@ -2,5 +2,4 @@
 <resources>
   <dimen name="enter_text_view_padding_start">120dp</dimen>
   <dimen name="enter_text_view_padding_end">120dp</dimen>
-  <integer name="profile_chooser_span_count">5</integer>
```

Comment:
Why was this removed?

Reply:
It was moved to instead `values-land-xhdpi/dimens.xml` of `values-xhdpi/dimens.xml` because we only update the span count based on the density in the landscape mode only.